iati_dashboard/api/views.py

```diff
@@ -13,6 +15,8 @@ class ReportingOrgViewSet(viewsets.ReadOnlyModelViewSet):
 
     queryset = ReportingOrg.objects.defer("stats_json").all().order_by("human_readable_name")
     serializer_class = ReportingOrgSerializer
+    filter_backends = (DjangoFilterBackend,)
```

Comment:
Is this just set up ready to use the filter, as opposed to actually using it now? Am I missing something about how `filter_backends` and `filterset_class` are used by `queryset`?